roles/overpass-api/tasks/set_up_database.yml

```diff
@@ -0,0 +1,15 @@
+- name: install osmctools
+  apt:
+    pkg: osmctools
+
+- name: download planet pbf
+  get_url:
+    url: "{{ planet_pbf_url }}"
+    dest: "{{ database_dir }}/planet-latest.osm.pbf"
+  become: true
+  become_user: "{{ user }}"
+
+- name: import planet
+  shell: osmconvert {{ database_dir }}/planet-latest.osm.pbf --out-osm | update_database --db-dir={{ database_dir }} --meta
+  become: true
+  become_user: "{{ user }}"
```

Comment:
You might want use the cloning feature and download the current database from dev.overpass-API.de/clone instead.